search/search_service/proxy/es_mappings.py

```diff
@@ -0,0 +1,75 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from elasticsearch_dsl import (
+    Date, Document, Keyword, Long, Text, normalizer,
+)
+
+column_names_normalizer = normalizer(name_or_instance='column_names_normalizer',
+                                     type='custom',
+                                     filter=['lowercase'])
+lowercase_normalizer = normalizer(name_or_instance='lowercase_normalizer',
+                                  type='custom',
+                                  char_filter=[],
+                                  filter=['lowercase', 'asciifolding'])
+feature_lowercase_normalizer = normalizer(name_or_instance='lowercase_normalizer',
+                                          type='custom',
+                                          char_filter=[],
+                                          filter=['lowercase', 'asciifolding'])
+
+
+class Table(Document):
```

Comment:
can you remind me why moving this here is a good idea

Reply:
we are not moving it the es mappings we use for databuilder are still in common, I can move this there too

Reply:
do not use dsl library to update, since we are not using documents for anything else this is really heavy weight